flocker/apiclient/test/test_client.py

```diff
@@ -403,16 +405,17 @@ def test_list_leases(self):
                 ])
             d.addCallback(lambda _: self.client.release_lease(d2))
             d.addCallback(lambda _: self.client.list_leases())
+            d.addCallback(frozenset)
             d.addCallback(
-                self.assertItemsEqual,
-                [
+                self.assertEqual,
```

Comment:
Newer Python implementations of `assertItemsEqual` rely on ordering being implemented consistently. `PClass` subclasses don't do that.